src/core/Footer/Footer.tsx

```diff
@@ -8,6 +8,7 @@ import { useTheme, StylingProps } from '../utils';
 import '@itwin/itwinui-css/css/footer.css';
 
 export type TitleTranslations = {
+  copyright: string;
```

Comment:
I don't think this `copyright` should be in TitleTranslations. Or at least it shouldn't be required.

We can still keep it in footerElements by changing the type of key to `keyof TitleTranslations | 'copyright' | (string & Record<never, never>)`

@veekeys What do you think?

Reply:
oh wait, right... Now users who use `TitleTranslations` will get this as breaking change!
Good catch, somehow missed it..
If they want to "translate" copyright, they most probably will use that through custom elements, cause not much to translate in copyright.
Lets remove that from here.

Reply:
Fixed? I think